FlowCrypt/Functionality/DataManager/Encrypted Storage/EncryptedStorage.swift

```diff
@@ -272,6 +272,18 @@ extension EncryptedStorage {
 }
 
 extension EncryptedStorage {
+    static var path: String {
+        get throws {
+            try getDocumentDirectory() + "/" + Constants.encryptedDbFilename
+        }
+    }
+
+    static var isStorageExists: Bool {
```

Comment:
let's call it `doesStorageFileExist`. This re-evaluates on each call, right?

Reply:
Yes, it re-checks storage file on each call